process/process.go

```diff
@@ -263,12 +263,14 @@ func (sp *systemProcess) GetMappings() ([]Mapping, uint32, error) {
 		}
 	}
 
-	fileToMapping := make(map[string]*Mapping, len(mappings))
-	for idx := range mappings {
-		m := &mappings[idx]
-		fileToMapping[m.Path.String()] = m
+	sp.fileToMapping = make(map[string]*Mapping)
```

Comment:
```suggestion
	fileToMapping := make(map[string]*Mapping)
```
Could we keep this original code and temporary variable?
It looks a bit cleaner, and helps with the thinking that `sp.fileToMapping` is replaced in one assignment.

Reply:
Ping?